uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -378,6 +378,29 @@ _Response Fields_
   Added in UAA 3.3.0
 </aside>
 
+When an access token is revoked using this endpoint, the UAA Introspect Token endpoint (``/introspect``)
+will respond with ``"active": false`` when presented with the revoked access token.
+
+If the access token is in the JWT format (as opposed to the opaque format), the server config ``uaa.jwt.revocable`` or
+the Identity Zone config ``config.tokenPolicy.jwtRevocable`` must be set to ``true`` for
+this feature to work. However, OAuth resource servers are generally not required to call the UAA Introspect
+Token endpoint to validate the status of the token. Once issued, a valid access token
+in the JWT format is generally considered valid until its expiry. Hence, we do not recommend
+relying on this endpoint to revoke access tokens in the JWT format. If the ability
+to remove/limit access after the tokens are issued is important to you, we recommend the following instead:
+
+* Ask the OAuth client to use opaque tokens only, so that the OAuth resource server is required to use
+the UAA Introspect Token endpoint to validate that the tokens have not been revoked.
```

Comment:
I personally understand this sentence, though I feel like it could confuse novice users through its length and complexity.

Reply:
Point discussed and/or edits made.